react/private/themeVars.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file was added as a short-term solution to using theme variables within javascript.

Reply:
To simplify the build tool requirements (e.g. to maintain webpack v1 support), this might be better off as a JS module rather than JSON.